tests/unit/file_io/test_file_io.py

```diff
@@ -1,7 +1,7 @@
 from letra._file_io.file_io import (
-    read_templates_from_file,
     get_path_for_read,
     get_path_for_write,
+    read_templates_from_file,
```

Comment:
Not directly related, just had this reordered locally to be alphabetic and included with this PR